src/test/java/org/gridsuite/gateway/TokenValidationTest.java

```diff
@@ -138,6 +166,31 @@ public void gatewayTest() throws Exception {
                 .jsonPath("$[0].studyName").isEqualTo("CgmesStudy")
                 .jsonPath("$[1].studyName").isEqualTo("IIDMStudy");
 
+        //Test a websocket with token in query parameters
+        WebSocketClient client = new StandardWebSocketClient();
+        client.execute(
+                URI.create("ws://localhost:" + this.localServerPort + "/notification/notify?access_token=" + token),
+            ws -> ws.receive().then()).subscribe();
+        // Busy loop waiting to check that spring-gateway contacted our wiremock server
+        // Is there a better way to wait for wiremock to complete the request ?
+        boolean done = false;
+        for (int i = 0; i < 100; i++) {
+            Thread.sleep(10);
```

Comment:
Shouldn't we remove that?

Reply:
I don't know how to remove it, if you can find a way go ahead